src/content/docs/apm/agents/c-sdk/instrumentation/distributed-tracing-c-sdk.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Distributed tracing for your C services'
+tags:
+  - Agents
+  - C SDK
+  - Instrumentation
+metaDescription: 'For the New Relic C SDK: how to implement distributed tracing across your C language applications and services.'
+redirects:
+  - /docs/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+  - /docs/apm/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+---
+
+import logsInContextTraces from 'images/logs-in-context-traces.png'
+
+import clogo from 'images/clogo.png'
+
+Distributed tracing allows you to see the entire journey of your requests throughout a [distributed system](/docs/distributed-tracing/concepts/introduction-distributed-tracing). The C SDK supports [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) with head-based sampling. It doesn't support the advanced feature called Infinite Tracing or W3C Trace context.
```

Comment:
I think "C SDK" should have a hyperlink to the actual repo: https://github.com/newrelic/c-sdk